x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.tsx

```diff
@@ -286,13 +286,29 @@ export const AddExceptionModalWrapper: React.FC<AddExceptionModalWrapperProps> =
     }
   }, [data?.hits.hits, isLoadingAlertData]);
 
+  /**
+   * This should be re-visited after UEBA work is merged
+   */
+  const useRuleIndices = useMemo(() => {
```

Comment:
Like @dplumlee mentioned, I might rename this since `useXXX` is the pattern for naming custom hooks, but not a biggy here.